app/views/lead_providers/your_schools/index.html.erb

```diff
@@ -0,0 +1,11 @@
+<%= render ::SchoolInformation::YourSchools::HeadingComponent.new %>
+<%= render ::SchoolInformation::YourSchools::NavigationComponent.new(cohorts: @cohorts, selected_cohort: @selected_cohort) %>
+<%= render ::SchoolInformation::YourSchools::ParticipantsComponent.new %>
```

Comment:
this seems extremely verbose, does components being nested this deeply improve overall organisation at all?

Reply:
I don't think it is very verbose as each component is small self contained piece. As for nesting until we have more examples for components and its reusability I would keep it for now as its very specific to Lead providers. I believe we need more examples of components to make an informed decision